src/ziggurat/init.clj

```diff
@@ -152,12 +176,20 @@
    {s/Keyword {:handler-fn (s/pred #(fn? %))
                s/Keyword   (s/pred #(fn? %))}}))
 
-(defn validate-stream-routes [stream-routes modes]
+(s/defschema BatchRoute
```

Comment:
This is not DRY. The entire thing is present in StreamRoute as well.

Reply:
There's a small difference: BatchRoute schema does not support channels while StreamRoute does.